kzg_prover/benches/kzg.rs

```diff
@@ -261,75 +258,65 @@ fn bench_kzg<
 }
 
 fn criterion_benchmark(_c: &mut Criterion) {
-    const N_CURRENCIES: usize = 2;
-    const N_POINTS: usize = 3;
+    const N_CURRENCIES: usize = 1;
+    const N_POINTS: usize = N_CURRENCIES + 1;
 
     // Demonstrating that a higher value of K has a more significant impact on benchmark performance than the number of users
     #[cfg(not(feature = "no_range_check"))]
     {
         const K: u32 = 18;
-        const N_USERS: usize = 16;
+        const N_USERS: usize = 2usize.pow(K - 1) + 2usize.pow(K - 2) - 6; // Which is equal to (1 << 17) + (1 << 16) - 6
```

Comment:
Can you explain the value 2^(K-1)-2^(K-2)-6? Why in the case of K=17 you're using 2^(K-1)-6?

Reply:
I think the correct max value should be 2^K - 6.

Reply:
I'm unable to provide a clear explanation for the failure of the $2^K - 6$ benchmark. Attempting to run it resulted in an error: 'vk generation should not fail: NotEnoughRowsAvailable { current_k: 17 }'.

Reply:
This has led me to believe that the maximum value for `N_USER` might be within the range between $2^{(K - 1)}$ and $2^K - 6`.